client/src/components/Tags.tsx

```diff
@@ -9,7 +9,7 @@ import Typography from '@mui/material/Typography';
 /* Generate pre-formatted TagItems based on an array of provided tags */
 export default function GenerateTags(props: { tags: string[], space?: number }) {
   return (
-    <Stack direction='row' spacing={props.space ? props.space : 1} style={{ alignItems: 'center' }}>
```

Comment:
Change this back and just don't pass space props so it can default to 1. I think I did this because I needed custom spacing somewhere, although i forgot where